src/lib/components/loading/loading.module.scss

```diff
@@ -4,90 +4,250 @@
 
 .loading {
   @include themify() {
-    background: themed('loadingBlack');
+    background: themed('loadingWhite');
```

Comment:
@rin I am testing this in the Portal. I need to change this back to loadingBlack because if not I can't see the dark mask in the Portal. Just write this here so if you want to rollback you can do it..